tests/test_greedy_follower.py

```diff
@@ -0,0 +1,120 @@
+import glob
+import os
+import os.path as osp
+
+import numpy as np
+
+import habitat_sim
+import habitat_sim.bindings as hsim
+import pytest
+import tqdm
+
+base_dir = osp.abspath(osp.join(osp.dirname(__file__), ".."))
+
+test_navmeshes = [
+    osp.join(base_dir, "data/scene_datasets/mp3d/17DRP5sb8fy/17DRP5sb8fy.navmesh"),
```

Comment:
This mp3d scene isn't in the test scenes archive, should we keep it, or just use the other ones for this test?

Reply:
The test will skip if it doesn't find the navmesh :)